scripts/analysis/amwg_table.py

```diff
@@ -206,8 +206,9 @@ def amwg_table(adf):
                 continue
             #End if
 
-            #Load model data from file:
-            data = _load_data(ts_files[0], var)
+            #Load model variable data from file:
+            ds = pf.load_dataset(ts_files)
```

Comment:
Do we want to only grab the first entry in `ts_files` here, or was the original way incorrect for using `ts_files[0]`?

Also with this change can we now delete the `_load_data` function from this python file?

Reply:
It should still be good. The old way of reading in files in this script was just `open_dataset` so the file needed to be pulled from the glob list. In the new `load_dataset` in `plotting_functions`, it has that built in in the function (your comment just before this one).

Reply:
Right!  That makes sense.  Thanks for the explanation!